patroni/ctl.py

```diff
@@ -445,6 +445,31 @@ def parse_scheduled(scheduled):
     return None
 
 
+@ctl.command('reload', help='Reload cluster member configuration')
+@click.argument('cluster_name')
+@click.argument('member_names', nargs=-1)
+@click.option('--role', '-r', help='Reload only members with this role', default='any',
+              type=click.Choice(['master', 'replica', 'any']))
+@option_force
+@click.pass_obj
+def reload(obj, cluster_name, member_names, force, role):
+    cluster = get_dcs(obj, cluster_name).get_cluster()
+
+    members = get_members(cluster, cluster_name, member_names, role, force, 'reload')
+
+    content = {}
+    for member in members:
+        r = request_patroni(member, 'post', 'reload', content, auth_header(obj))
+        if r.status_code == 200:
+            click.echo('Success: reload on member {0}'.format(member.name))
+        elif r.status_code == 202:
+            click.echo('Success: reload scheduled on member {0}'.format(member.name))
```

Comment:
`POST /reload` and `SIGHUP` are updating postgresql.conf and reloading it not immediately, but within `loop_wait` seconds. Just wondering, does it make sense to change this status message, to avoid confusion with `scheduled restart`, where meaning of `scheduled` is absolutely different?

Reply:
Yes you are right. I was a little unsure of this myself but figured to start the PR and sort it out then.

So if code is 200, echo something to the effect of no changes to apply? Or just echo nothing?

And if 202, do we not say reload scheduled? Can we say that the reload finished if we don't know that?

Reply:
202 means that there is something has changed in the config and eventually (within `loop_wait`) Patroni will apply them. Actually, it will not necessary case update of postgresql.conf and pg_ctl reload.

Reply:
> it will not necessary case update of postgresql.conf and pg_ctl reload.

I'm not sure what you're saying here. 

What wording/language do we want to use here to tell the user? Something like "Reload request received and will be processed in loop_wait seconds"?

Reply:
> I'm not sure what you're saying here.

One can update `tags` in patroni.yaml, and this update will not cause pg_ctl reload, because nothing in postgresql.conf was changed.


> "Reload request received and will be processed in loop_wait seconds" 

sounds good.